LICENSES/LICENSE

```diff
@@ -0,0 +1,207 @@
+================================================================================
+= KubeSphere licensed under: =
```

Comment:
This format looks weird, why is there a `:`

Reply:
The script is copied from https://github.com/kubernetes/kubernetes/blob/master/hack/update-vendor-licenses.sh. The license directory is helpful for us.

Reply:
Got it.